templates/ansible/facts.erb

```diff
@@ -27,20 +29,20 @@ module: <%= module_name(object) %>_facts
 description:
   - Gather facts for GCP <%= object.name %>
 short_description: Gather facts for GCP <%= object.name  %>
-version_added: <%= lines(@config.manifest.get('version_added', config)) -%>
-author: <%= lines(@config.manifest.get('author', config)) -%>
+version_added: <%= lines(@config.manifest.get('version_added', object)) -%>
+author: <%= lines(@config.manifest.get('author', object)) -%>
 requirements:
-<% @config.manifest.get('requirements', config).each do |line| -%>
+<% @config.manifest.get('requirements', object).each do |line| -%>
 <%= lines(indent(bullet_line(line, 4, false, false), 4)) -%>
 <% end -%>
 options:
     filters:
        description:
            A list of filter value pairs. Available filters are listed here
-           U(<%= object.filter_url -%>).
+           U(https://cloud.google.com/compute/docs/labeling-resources#filter).
```

Comment:
https://cloud.google.com/sdk/gcloud/reference/topic/filters might be a better resource for this, since it's not specific to labels.

Reply:
Great find. I like that link a lot more.